backend/kesaseteli/applications/migrations/0001_initial.py

```diff
@@ -8,7 +8,6 @@
 
 
 class Migration(migrations.Migration):
```

Comment:
Migrations should have been excluded from ruff formatting per the configurations in pyproject.toml, so they should not come up as changes.